addons/web/controllers/main.py

```diff
@@ -1030,6 +1030,22 @@ def test_mobile_suite(self, mod=None, **kwargs):
     def benchmarks(self, mod=None, **kwargs):
         return request.render('web.benchmark_suite')
 
+    @http.route('/web/bundle/<string:bundle_name>', type='http', auth="user", methods=["GET"])
+    def bundle(self, bundle_name):
+        """
+        Request the definition of a bundle, including its javascript and css bundled assets
+        One might think that making this controller type=json is a good idea, but in fact odoo cannot receive Get
+        requests for routes with type=json that has no body, and the browser's fetch API cannot generate a Get request
+        that has a body.
+        """
+        all = request.env["ir.qweb"]._get_asset_nodes(bundle_name, js=True, css=True, async_load=True)
```

Comment:
No need to specify `async_load=True` since you're ignoring the corresponding value in the returned `attrs` dict.

Reply:
No, we need this so the assets generated are async at the top level function, so the top level modules can be async too

Reply:
I'm not entirely sure what you mean by "top-level function" in this context, but odoo modules declared with the ES-module syntax are always declared as async functions, this parameter controls whether the script node emitted during server-side qweb compilation will have the async `<script>` attribute (eg, whether the "node" that's returned should have `async` in the attrs dict).